src/server/api/routers/files.ts

```
@@ -1,536 +1,573 @@
+              transferStatus: "idle",
+            },
+          });
+      }
+
+      // Delete the file from the corresponding service
+      switch (file.storedIn) {
+        case "utfs":
+          if (!file.ufsKey) {
+            throw new Error("No Key Provided");
+          }
+          const deletionResponse = await utapi.deleteFiles(file.ufsKey);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)